packages/integration-tests/projects/suite-web/tests/suite/safety-checks-warning.test.ts

```diff
@@ -69,17 +93,33 @@ describe('safety_checks Warning For PromptTemporarily', () => {
     });
 
     it('Dismissed warning re-appears when safety_checks are set to strict and then to Prompt again.', () => {
+        cy.getTestElement('@settings/device/safety-checks-button')
+            .scrollIntoView({ offset: { top: -150, left: 0 } })
+            .click({ scrollBehavior: false });
+        cy.get(`[data-test="@radio-button-prompt"]`).click();
+        cy.getTestElement('@safety-checks-apply').click();
+        cy.task('pressYes');
+        cy.wait(666);
         // Dismiss the warning.
         cy.getTestElement('@banner/safety-checks/dismiss').click();
         // Open the safety_checks setting modal and change safety_checks to Strict.
-        cy.getTestElement('@settings/device/safety-checks-button').click();
+        cy.getTestElement('@settings/device/safety-checks-button')
+            .scrollIntoView({ offset: { top: -150, left: 0 } })
+            .click({
+                scrollBehavior: false,
+            });
```

Comment:
I'm wondering if this would work as well? 

```suggestion
            .click({scrollBehavior: 'bottom'});
```

Reply:
works for me, replaced this and `{ force: true }` everywhere I used id, now waiting for CI.